src/test/java/seedu/address/logic/parser/SortCommandParserTest.java

```diff
@@ -48,52 +61,45 @@ public void parse_validArguments_success() throws ParseException {
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "phone", "asc");
-        actualCommand = parser.parse("person phone asc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "phone", "desc");
-        actualCommand = parser.parse("person phone desc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "email", "asc");
-        actualCommand = parser.parse("person email asc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "email", "desc");
-        actualCommand = parser.parse("person email desc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "address", "asc");
-        actualCommand = parser.parse("person address asc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
-
-        expectedCommand = new SortCommand("person", "address", "desc");
-        actualCommand = parser.parse("person address desc");
-
-        assertEquals(expectedCommand.getField(), actualCommand.getField());
-        assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
+        // For person sorts
+        String list = ACCEPTED_LIST_PARAMETERS.get(0);
+        for (String field: ACCEPTED_FIELD_PARAMETERS.subList(0, 4)) {
+            for (String order : ACCEPTED_ORDER_PARAMETERS) {
+                expectedCommand = new SortCommand(list, field, order);
+
+                // Commands for full sort command word
+                actualCommand = parser.parse(list + " " + field + " " + order);
+                assertEquals(expectedCommand.getList(), actualCommand.getList());
+                assertEquals(expectedCommand.getField(), actualCommand.getField());
+                assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
+
+                // Commands for sort command alias
+                actualCommand = parser.parse(list + " " + field + " " + order);
+                assertEquals(expectedCommand.getList(), actualCommand.getList());
+                assertEquals(expectedCommand.getField(), actualCommand.getField());
+                assertEquals(expectedCommand.getOrder(), actualCommand.getOrder());
```

Comment:
Instead of using multiple assertEquals, can we use 1 assertEquals(expectedCommand, actualCommand) to do the same thing?